docs/.vuepress/sidebar/en.ts

```diff
@@ -935,14 +935,34 @@ export const enSidebar = sidebar({
       text: "Running a Hyperchain",
       collapsible: true,
       children: [
+        {
+          text: "Overview",
+          link: "/zk-stack/running-a-hyperchain/"
+        },
         {
           text: "Locally",
```

Comment:
This menu title doesn't make sense. Need to think of something that encapsulates locally + in prod

Reply:
and the subpath should match whatever name your think of